### ISSUE TITLE
Support for VP9 hardware decoding at 4K resolution & 60 fps?

### ISSUE THREAD

**Issue by Hedda** [koying/SPMC#312]
Hi Koying, can you maybe confirm if SPMC supports VP9 hardware decoding at 2160p resolution & 60 frames per second?

See example with sample links:

http://www.cnx-software.com/2016/06/07/4k-vp9-video-playback-in-amlogic-s905x-based-nexbox-a5-android-tv-box/

4K VP9 @ 60 fps should on paper be support by both Nvidia Shield TV & the new Amlogic S905X SoC but does SPMC?

**Comment by war59312** [comment]
tara-no9-vp9.webm plays back perfectly for me on my Nvidia Shield TV. 👍 

In 4k and zero dropped frames. 💃 

Too bad no subtitles and they are clearly lip singing.

Edit: Crazy how this damn box is more powerful video wise than my computer.

Granted, my computer is old now (2011). My computer can't make it past 19 seconds. lol

That's with MPC-HC. VLC can't play it at all.

**Comment by Hedda** [comment]
@war59312 yes but is that software decoded (on CPU) or hardware decoded with the VPU circuits integrated in the GPU?

Please note that this question/request is for hardware decoded with the VPU circuits integrated in the GPU, and not software decoded on CPU.

**Comment by koying** [comment]
It's h/w decoded

**Comment by Hedda** [comment]
For reference, here is URLs to some more specific examples which have 4K VP9 videos:

https://www.youtube.com/watch?v=s6zR2T9vn2c
https://www.youtube.com/watch?v=XCbir1vyUA0
https://www.youtube.com/watch?v=doa9asyxlMU
https://www.youtube.com/watch?v=I7CycuMpQ3w
https://www.youtube.com/watch?v=UceRgEyfSsc

You can use the youtube-dl python script ( https://rg3.github.io/youtube-dl/ ) find out if as video (URL) is available in VP9 2160p format and download it:

youtube-dl URL --list-formats

Then download it using the URL together with the format code:

youtube-dl URL -f format_code
